### ISSUE TITLE
HardNet training

### ISSUE THREAD

**Issue by amitshaked** [rmitra/PS-Dataset#4]
Hey,

Can you please upload your pytorch code for processing the dataset and use it to train the network with HardNet loss? 

Thanks!

**Comment by Shu-HowTing** [comment]
+1

**Comment by rmitra** [comment]
We used Torch7 framework in our implementation. I will recommend to refer to https://github.com/DagnyT/hardnet for the most efficient pytorch implementation.
